<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#2C2E2F;
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="ProjectName">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">    color:#c1c1c1;
    background-color:#2C2E2F;
    border-radius:8px;
    border: 1px solid #2C2E2F;
    border-bottom: 1px solid #27292a;
	padding: 2px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="OpenInterpreter">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton{
    border-style: solid;
    border-width: 1px;
    border-color: #27292a;
    color:#c1c1c1;
	background-color:#414243;
	padding-left: 5px;
}
QToolButton::pressed {
    background-color: #6b6c6d;
    border-style: inset;
}</string>
            </property>
            <property name="text">
             <string>Открыть
интерпретатор</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="CheckErrors">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton{
    border-style: solid;
    border-width: 1px;
    border-color: #27292a;
    color:#c1c1c1;
	background-color:#414243;
	padding-left: 5px;
}
QToolButton::pressed {
    background-color: #6b6c6d;
    border-style: inset;
}</string>
            </property>
            <property name="text">
             <string>Проверить код
на ошибки</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="EnterCodeInConsole">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton{
    border-style: solid;
    border-width: 1px;
    border-color: #27292a;
    color:#c1c1c1;
	background-color:#414243;
	padding-left: 5px;
}
QToolButton::pressed {
    background-color: #6b6c6d;
    border-style: inset;
}</string>
            </property>
            <property name="text">
             <string>Ввести код 
в консоль</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>13</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">color: #c1c1c1;</string>
            </property>
            <property name="text">
             <string>Discord:
Alleksh#0284</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">color:#c1c1c1;</string>
              </property>
              <property name="text">
               <string>VK: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="VKDirectButton">
              <property name="styleSheet">
               <string notr="true">QPushButton {
color: lightblue;
    border: none;
}

QPushButton:pressed {

}

QPushButton:flat {
}

QPushButton:default {

}</string>
              </property>
              <property name="text">
               <string>vk.com/alleksh</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QTabWidget" name="Files">
        <property name="styleSheet">
         <string notr="true">QTabWidget::pane { /* The tab widget frame */

    border-top: 2px solid #007ACC;
	border-bottom: 1px solid #232425;
	border-left:1px solid #232425;
}

QTabWidget::tab-bar {
    left: 5px; /* move to the right by 5px */
}

/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
background-color:#4a4d4f;
 border: 2px solid #4a4d4f;
    border-bottom-color: #4a4d4f; /* same as the pane color */
    min-width: 12px;
    min-height: 12px;
    padding: 4px;
    padding-top: 2px;
	padding-bottom:02px;
}

QTabBar::tab:hover {

    background-color: #6b6c6d;
	border:#6b6c6d;
}

QTabBar::tab:selected {
    border-color: #007ACC;
    color:#FAFAFA;
	background:#007ACC;
}

QTabBar::tab:!selected {
color:#c1c1c1;
}
QTabBar::close-button {
image: url(:/images/close.png); padding: 3px;
} 
QTabBar::close-button:hover { image: url(:/images/close-hover.png); padding: 3px; }</string>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="iconSize">
         <size>
          <width>4</width>
          <height>4</height>
         </size>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
        <property name="movable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="tab_3">
         <property name="styleSheet">
          <string notr="true">
QTabBar::close-button { image: url(qrc:/close.jpg) subcontrol-position: left; } 
QTabBar::close-button:hover { image: url(qrc:/close-hover.jpg) }</string>
         </property>
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_4">
         <property name="styleSheet">
          <string notr="true">
QTabBar::close-button { image: url(qrc:/close.jpg) subcontrol-position: left; } 
QTabBar::close-button:hover { image: url(qrc:/close-hover.jpg) }</string>
         </property>
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QTabWidget" name="CompilerOutput">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QTabWidget::pane { /* The tab widget frame */

    border-top: 2px solid #007ACC;
	border-bottom: 1px solid #232425;
	border-left:1px solid #232425;
}

QTabWidget::tab-bar {
    left: 5px; /* move to the right by 5px */
}

/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
background-color:#4a4d4f;
 border: 2px solid #4a4d4f;
    border-bottom-color: #4a4d4f; /* same as the pane color */
    min-width: 12px;
    min-height: 12px;
    padding: 4px;
    padding-top: 2px;
	padding-bottom:02px;
}

QTabBar::tab:hover {

    background-color: #6b6c6d;
	border:#6b6c6d;
}

QTabBar::tab:selected {
    border-color: #007ACC;
    color:#FAFAFA;
	background:#007ACC;
}

QTabBar::tab:!selected {
color:#c1c1c1;
}
QTabBar::close-button {
image: url(:/images/close.png); padding: 3px;
} 
QTabBar::close-button:hover { image: url(:/images/close-hover.png); padding: 3px; }</string>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <property name="tabsClosable">
         <bool>false</bool>
        </property>
        <property name="movable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Проблемы</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QTextBrowser" name="TextAnalysisOutput">
            <property name="styleSheet">
             <string notr="true">QScrollBar:horizontal {
    color:#c1c1c1;
background-color:#2C2E2F;
}

QScrollBar::handle:horizontal {

 border: 2px groove #6b6c6d;
border-radius:4px;
background-color:#2C2E2F;
    color:#c1c1c1;
    min-width: 20px;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
    color:#c1c1c1;
background-color:#2C2E2F;
}

QScrollBar:vertical {    
color:#c1c1c1;
background-color:#2C2E2F;
 }
 QScrollBar::handle:vertical {
 border: 2px groove #6b6c6d;
border-radius:4px;
background-color:#2C2E2F;
    color:#c1c1c1;
     min-height: 20px;
 }
 QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: #2C2E2F;
 }

QTextBrowser
{
 border-style: solid;
    border-width: 2px;
    border-color: #27292a;
    color:#c1c1c1;
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Вывод анализатора проекта</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTreeView" name="ProjectTree">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QTreeView {
background-color:#414243;
border:1px solid #414243;
}

QTreeView::item {
    color:#c1c1c1;
}

QTreeView::item:hover {
    background-color: #6b6c6d;
    border: 1px solid #232425;
}

QTreeView::item:selected {
    border: 1px solid #232425;
background-color:#27292a;
}

QTreeView::item:selected:active{
    background-color: #6b6c6d;
}

QTreeView::item:selected:!active {
    background-color: #6b6c6d;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar::item {
color:#c1c1c1;
    spacing: 8px; /* spacing between menu bar items */
    padding: 3px 6px;
    background: transparent;
}

QMenuBar::item:selected { /* when selected using mouse or keyboard */
    background: #808182;
}

QMenuBar::item:pressed {
    background: #565758;
}</string>
   </property>
   <widget class="QMenu" name="Project">
    <property name="styleSheet">
     <string notr="true">QMenu {
    background-color: #414243; /* sets background of the menu */
    border: 1px solid #232425;
}

QMenu::item {
    /* sets background of menu item. set this to something non-transparent
        if you want menu color and menu item color to be different */
    background-color: transparent;
    color:#c1c1c1;
    padding: 4px 8px;
}

QMenu::item:selected { /* when user selects item using mouse or keyboard */
    background-color: #6b6c6d;
}</string>
    </property>
    <property name="title">
     <string>Проект</string>
    </property>
    <addaction name="OpenProject"/>
    <addaction name="CreateProject"/>
    <addaction name="separator"/>
    <addaction name="SaveAndExit"/>
   </widget>
   <widget class="QMenu" name="WordList">
    <property name="styleSheet">
     <string notr="true">QMenu {
    background-color: #414243; /* sets background of the menu */
    border: 1px solid #232425;
}

QMenu::item {
    /* sets background of menu item. set this to something non-transparent
        if you want menu color and menu item color to be different */
    background-color: transparent;
    color:#c1c1c1;
    padding: 4px 8px;
}

QMenu::item:selected { /* when user selects item using mouse or keyboard */
    background-color: #6b6c6d;
}</string>
    </property>
    <property name="title">
     <string>Словарь</string>
    </property>
    <addaction name="EditCurrentWordList"/>
    <addaction name="LoadStandardWordList"/>
   </widget>
   <widget class="QMenu" name="Help">
    <property name="styleSheet">
     <string notr="true">QMenu {
    background-color: #414243; /* sets background of the menu */
    border: 1px solid #232425;
}

QMenu::item {
    /* sets background of menu item. set this to something non-transparent
        if you want menu color and menu item color to be different */
    background-color: transparent;
    color:#c1c1c1;
    padding: 4px 8px;
}

QMenu::item:selected { /* when user selects item using mouse or keyboard */
    background-color: #6b6c6d;
}</string>
    </property>
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="WordListHelp"/>
    <addaction name="InterfaceHelp"/>
    <addaction name="FORTHHelp"/>
    <addaction name="SpecSymbolsHelp"/>
   </widget>
   <addaction name="Project"/>
   <addaction name="WordList"/>
   <addaction name="Help"/>
  </widget>
  <widget class="QStatusBar" name="Status">
   <property name="styleSheet">
    <string notr="true">
QStatusBar {
background:#007ACC;
color:#fafafa;
}</string>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="EditCurrentWordList">
   <property name="text">
    <string>Отредактировать словарь</string>
   </property>
  </action>
  <action name="LoadStandardWordList">
   <property name="text">
    <string>Загрузить стандартный словарь</string>
   </property>
  </action>
  <action name="WordListHelp">
   <property name="text">
    <string>Помощь по взаимодействию со словарем</string>
   </property>
  </action>
  <action name="InterfaceHelp">
   <property name="text">
    <string>Помощь по интерфейсу</string>
   </property>
  </action>
  <action name="FORTHHelp">
   <property name="text">
    <string>Помощь по FORTH</string>
   </property>
  </action>
  <action name="SpecSymbolsHelp">
   <property name="text">
    <string>Помощь по спец. символам</string>
   </property>
  </action>
  <action name="OpenProject">
   <property name="text">
    <string>Открыть проект</string>
   </property>
  </action>
  <action name="CreateProject">
   <property name="text">
    <string>Создать проект...</string>
   </property>
  </action>
  <action name="SaveAndExit">
   <property name="text">
    <string>Сохранить и выйти</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
